compiler/test/fail_compilation/test24353.d

```diff
@@ -0,0 +1,24 @@
+// https://issues.dlang.org/show_bug.cgi?id=24353
+
+/**
+TEST_OUTPUT:
+---
+fail_compilation/test24353.d(21): Error: mutable method `test24353.S.opApply` is not callable using a `const` object
+fail_compilation/test24353.d(12):        Consider adding `const` or `inout` here
```

Comment:
Looks like you forgot to update the line numbers after adding the bugzliia link. Should be 23 & 14